subsys/bluetooth/controller/ll_sw/ull_conn_internal.h

```diff
@@ -7,17 +7,52 @@
 /* Macro to convert time in us to connection interval units */
 #define RADIO_CONN_EVENTS(x, y) ((u16_t)(((x) + (y) - 1) / (y)))
 
-/* Macro to return PDU time */
+/*
+ * Macros to return correct PDU time
+ * Note: formula is valid for S=8 coding only
```

Comment:
Formula is valid for 1M, 2M and Coded S8.
It's confusing to say it is valid for S8 coded only. Because it supports the other ones.